pkg/util/files.go

```diff
@@ -232,3 +232,23 @@ func CopyDir(src string, dst string) error {
 
 	return nil
 }
+
+// FindFiles walks a given directory and returns a list of file names containing the provided search string.
+func FindFiles(dirpath string, search string) ([]string, error) {
```

Comment:
Please allow globbing or regex, it will make this a far better utility function. I'd think we could import something that would be more complete than what we can write here though?